manager2/src/app/admin/projects/projects.component.html

```diff
@@ -1,7 +1,7 @@
   <div *ngIf="notification" class="alert alert-success">{{notification}}</div>
   <div class="card bg-light">
     <div class="card-header" (click)="requests_visible=!requests_visible" style="cursor: pointer;">
-      <h3>{{requests_number}} request(s) waiting </h3>
+      <h3>{{requests_number + pending_number}} request(s) waiting </h3>
```

Comment:
separated the number of requests from the number of pending projects. Underneath that is the added UI elements allowing their management